src/sentry/sentry_metrics/indexer/postgres/postgres_v2.py

```diff
@@ -409,6 +409,27 @@ def reverse_resolve(self, use_case_id: UseCaseID, org_id: int, id: int) -> Optio
         string: str = obj.string
         return string
 
+    def bulk_reverse_resolve(
+        self, use_case_id: UseCaseID, org_id: int, ids: Collection[int]
+    ) -> List[Optional[str]]:
+        """Looks up the stored strings for the given ids.
+
+        Returns a list with the same length as ids parameter. The at position n the result contains
+        either the resolved string for ids[n] or None if ids[n] cannot be resolved.
+        """
+        metric_path_key = METRIC_PATH_MAPPING[use_case_id]
+        table = self._get_table_from_metric_path_key(metric_path_key)
+        try:
+            strings = table.objects.get_many_from_cache(id=id, use_replica=True)
```

Comment:
I can't seem to find where this `id` is created, maybe you meant:
```suggestion
            strings = table.objects.get_many_from_cache(id__in=ids, use_replica=True)
```

Reply:
yep that's wrong it should be:
`get_many_from_cache(ids)`

Reply:
If we want all the elements whose id match `ids` shouldn't we use `id__in=ids`?

Reply:
no, `get_many_from_cache` is more specialised, it doesn't allow normal django parameters.
It only gets a list of keys and optionally then name of the key column ( by default the primary key i.e. `"pk"`).